### PR TITLE
EmailDB

### PR DESCRIPTION
**Pull request by Reticentman**
Added emails to DB and to the views.

### CODE REVIEW

File: deploy/insert_user.sql [hunk 1]
```diff
@@ -7,7 +7,9 @@ BEGIN;
 CREATE OR REPLACE FUNCTION upsapp.insert_user(
     nickname  TEXT,
     password  TEXT,
-    available BOOLEAN
+    available BOOLEAN,
+    image     TEXT,
+    email     TEXT
```
Comment (bsima) [inline deploy/insert_user.sql]: This looks good, the only problem is that you are simply changing the `insert_user.sql` file when you should actually be adding a _new_ `.sql` file for these changes because each change (called a "database migration") to the database should only happen once.

Think about it like this: if you have a production database on a server somewhere, it needs to be serving the users, so you can't take it down to change the database whenever you want. This means that each change to the database must take the old database and "migrate" it to a new version. You can only add new migrations to the `deploy` directory, you can't modify a migration once you've created and deployed it because then you would have to go in and tamper with the existing user data. 

In short, once you deploy a database migration, you shouldn't change it. If you need to make a change to the table or whatever, just make a new migration `.sql` file.

In this case we don't have to worry because we don't have a production database yet, but its good to know this.
